src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -103,6 +103,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="..\src\System\Text\Json\BitStack.cs" Link="BitStack.cs" />
+    <Compile Include="WritableJsonApiTests.cs" />
```

Comment:
NIT: can we just move these two Compile items under the previous ItemGroup with the rest?